browser/build.js

```diff
@@ -78,12 +78,13 @@ async.waterfall([
       exec(cmd, cb)
     },
     function(stdout, stderr, cb) {
-      var cmd = 'gofmt -s -w -l bindata_assetfs.go'
+      var cmd = 'gofmt -s -w -l bindata.go'
       console.log('Running', cmd)
       exec(cmd, cb)
     },
     function(stdout, stderr, cb) {
-      fs.renameSync('bindata_assetfs.go', assetsFileName)
+      console.log("assetsFileName:", assetsFileName)
+      fs.renameSync('bindata.go', assetsFileName)
```

Comment:
I don't think we need any changes to this file. Can you please check?

Reply:
Done. Working on it in a time between other tasks being in review makes me forget about minor things, thank you for  keeping focus on clear code